### PR TITLE
addition of exstra var, galaxy_systemd_sub_dest to ensure multiple galaxy instances can run

### PR DESCRIPTION
**Pull request by Dirowa**
Addition of the variable galaxy_systemd_sub_dest

### CODE REVIEW

File: tasks/mule.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     group: root
     mode: 0644
     src: galaxy.service.j2
-    dest: /etc/systemd/system/galaxy.service
+    dest: /etc/systemd/system/{{ galaxy_systemd_sub_dest }}/galaxy.service
```
Comment (hexylena) [inline tasks/mule.yml]: I'm not sure this works, to put them in folders like this, both are still called `galaxy.service` which is the important part for systemd AFAIK. We can instead leave them at the same level and just make the name configurable for the same result.

```suggestion
    dest: /etc/systemd/system/{{ galaxy_systemd_unit_name }}.service
```

File: templates/galaxy-handler@.service.j2 [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Galaxy Handlers
+Description=Galaxy Handlers for {{galaxy_systemd_sub_dest}}
```
Comment (hexylena) [inline templates/galaxy-handler@.service.j2]: This is probably ok, we often call it an `galaxy_instance_codename` internally. Maybe something like
```suggestion
Description=Galaxy Handlers{% if galaxy_instance_codename %} for {{ galaxy_instance_codename | default(galaxy_systemd_unit_name) }}{% endif %}
```

File: tasks/mule.yml (hunk 1 above)
Comment (Dirowa) [reply tasks/mule.yml]: i agree, i will edit the changes

File: templates/galaxy-handler@.service.j2 (hunk 2 above)
Comment (Dirowa) [reply templates/galaxy-handler@.service.j2]: This will be cleaner yes

**Comment by Dirowa** [comment]
incorperated all review edits

**Comment by Dirowa** [comment]
all commentes are incorperated

**Review by hexylena — APPROVED** [review]
This looks good!

**Comment by Dirowa** [comment]
shall i also update the training material that this now an option?

**Comment by hexylena** [comment]
nah the training material won't need to cover it, it's way too special of a case

**Comment by hexylena** [comment]
Also we're deprecating that role soon, maybe with the next admin training. Instead we'll build the systemd portion into the Galaxy role and there it'll be done properly.

**Comment by Dirowa** [comment]
ahhh okay :) thank you for your help!

**Pull request merged**